File: php/db_connect.php
<?php
try {
    $pdo = new PDO(
        "mysql:host=localhost;dbname=classroom_finance_tracker;charset=utf8mb4",
        "root",
        ""
    );
    $pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
    $pdo->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
} catch (PDOException $e) {
    die(json_encode([
        "error" => "Database Connection Failed: " . $e->getMessage()
    ]));
}
?>

File: C#/db_connect.cs
using MySql.Data.MySqlClient;

public static class DbConnect
{
    public static MySqlConnection GetConnection()
    {
        // Update with your actual connection details
        var connectionString = "Server=localhost;Database=classroom_finance_tracker;Uid=root;Pwd=;";
        return new MySqlConnection(connectionString);
    }
}